docs/commands.md

```diff
@@ -716,7 +716,7 @@ MATCH p=()-[*]->() RETURN [node IN nodes(p) WHERE node.rank > 10 | node.name]
 ```
 
 #### Existential comprehension functions
-The functions `any()` and `all()` use a simplified form of the list comprehension syntax and return a boolean value.
+The functions `any()`, `all()`, `single` and `none` use a simplified form of the list comprehension syntax and return a boolean value.
```

Comment:
For consistency, either all function names should be followed by parentheses or none of them should be.

Reply:
Add documentation at line 611 as well.